railties/lib/rails/generators/app_base.rb

```diff
@@ -326,7 +326,7 @@ def convert_database_option_for_jruby
       def assets_gemfile_entry
         return [] if options[:skip_sprockets]
 
-        GemfileEntry.version("sass-rails", "~> 5.0", "Use SCSS for stylesheets")
+        GemfileEntry.version("sassc-rails", "~> 2.1", "Use SCSS for stylesheets")
```

Comment:
Should we remove the upper version bound here? Like we’ve done for other dependencies.